mysql-test/suite/sys_vars/t/binlog_ddl_skip_rewrite_basic.test

```diff
@@ -0,0 +1 @@
+SELECT @@global.binlog_ddl_skip_rewrite;
```

Comment:
Please add more scenarios. Check existing testcases for dynamic.
1. try session
2. set invalid values
3. set allowed values
4. try SET var= default (default is allowed value) and check variable value